packages/components/src/Tokens/TokensStyledComponents.tsx

```diff
@@ -101,25 +101,23 @@ export const FlexColumnsSecondaryModule = styled.div`
   }
 `;
 
-export const TokenHeaderOuter = styled.div`
+export const TokenHeader = styled.div`
+  color: ${colors.primary.BLACK};
   font-family: ${fonts.SANS_SERIF};
   padding: 30px 30px 40px;
   text-align: center;
-  width: 100%;
-`;
 
-export const TokenHeader = styled.h2`
-  color: ${colors.primary.BLACK};
-  font-size: 27px;
-  line-height: 39px;
-  margin-bottom: 25px;
-`;
+  h2 {
```

Comment:
I'm starting to really like using sub selectors in styled components 👍

Reply:
Yes! I think it makes it a little cleaner and easier to read